ddtrace/llmobs/_llmobs.py

```diff
@@ -409,7 +410,9 @@ def _llmobs_span_event(self, span: Span) -> Optional[LLMObsSpanEvent]:
 
     @staticmethod
     def _llmobs_tags(span: Span, ml_app: str, session_id: Optional[str] = None) -> List[str]:
+        dd_tags = parse_tags_str(os.environ.get("DD_TAGS"))
         tags = {
+            **dd_tags,
```

Comment:
we should be able to get it from `ddtrace.config.tags` to save parsing it every time we generate tags